### PR TITLE
Update according to Radium release-candidate

### PR DESCRIPTION
**Pull request by nmellado**
- [cmake] Use versionless qt packages

### CODE REVIEW

**Comment by nmellado** [comment]
@dlyr , didn't we have a system that pick the right branch on Radium-Engine repository, in the CI ?

**Comment by dlyr** [comment]
Nope, this is done in priave repo, i'll run the action.

File: .github/workflows/release-candidate-push-ci.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+name: Compile Radium Apps
```
Comment (nmellado) [inline .github/workflows/release-candidate-push-ci.yml:1]:
```suggestion
name: Compile Radium Apps (Release Candidate)
```

File: .github/workflows/release-candidate-push-ci.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+on:
+  push:
+    branches:
+      - release-candidate
+env:
+  ext-dir: ${GITHUB_WORKSPACE//\\//}/external/install/
+
+jobs:
+  build:
```
Comment (nmellado) [inline .github/workflows/release-candidate-push-ci.yml:15]:
```suggestion
  build-rc:
```

**Review by dlyr — DISMISSED** [review]

**Comment by MathiasPaulin** [comment]
Is this PR still of interest ? release-candidate is up to date with Radium release candidate.

**Comment by dlyr** [comment]
@MathiasPaulin I don't get your question. Do you want to close this PR without merging ?

**Comment by MathiasPaulin** [comment]
> @MathiasPaulin I don't get your question. Do you want to close this PR without merging ?

No, I haven't seen that this PR is from release-candidate to master and I was confused by the description of this PR. 
But I think we need first to update Radium master before this PR could be tested and merged.

**Review by MathiasPaulin — APPROVED** [review]

**Pull request merged**